geode-connectors/src/main/java/org/apache/geode/connectors/jdbc/internal/cli/CreateMappingCommand.java

```diff
@@ -324,12 +328,19 @@ public boolean commandAvailable() {
     return isOnlineCommandAvailable();
   }
 
-  private void alterRegion(String queueName, RegionAttributesType attributes, boolean synchronous) {
-    setCacheLoader(attributes);
-    if (synchronous) {
-      setCacheWriter(attributes);
+  private void alterRegion(String queueName, RegionAttributesType attributes, boolean synchronous,
```

Comment:
Instead of adding another boolean flag to this method; I think it would be better to split "alterRegion" into two methods: "alterProxyRegion" and "alterNonProxyRegion".